applications/EmpireApplication/test_examples/CoSim_DevExamples/oscilating_cylinder_fsi/ProjectParametersCFD.json

```diff
@@ -1,173 +1,233 @@
 {
-    "problem_data"                     : {
-        "problem_name"    : "CFD_Cylinder",
-        "domain_size"     : 2,
-        "parallel_type"   : "OpenMP",
-        "echo_level"      : 0,
-        "start_time"      : 0.0,
-        "end_time"        : 10
+    "problem_data": {
+        "problem_name": "CFD_Cylinder",
+        "parallel_type": "OpenMP",
+        "echo_level": 0,
+        "start_time": 0,
```

Comment:
why do you change this? This is supposed to be a double

Reply:
thanks for pointing out, I used a Json editor tool to guarantee correct structure of the file but it seems to truncate unnecessary decimals - I restored it in the last commit